modules/tictactoe/tictactoe.js

```diff
@@ -50,35 +58,35 @@ function findGame(thread, player) {
         if (game.noughts === player || game.crosses === player) {
             return game;
         }
-    } 
+    }
 }
 
 /**
  * Places a piece in a certain game
 */
 function place(game, player, move) {
-    //Work out what piece we are using.
+    // Work out what piece we are using.
     var piece = "X",
         board = game.board;
 
     if (player === game.noughts) {
         piece = "O";
     }
 
-    //It's not even your turn
+    // It's not even your turn
     if (player !== game.turn) {
-        return "Invalid Move! (it's not your turn " + player + ")";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/theprojectshop/Kassy/file/2529976290/issues/source?bid=3282987&fileBranchId=3282998#l78)